### DISCUSSION TITLE
Append parameters to a USDZ link thats been converted from a GLB

### DISCUSSION THREAD

**Discussion by UnityEQ** [google/model-viewer#2694] [Q&A]
So now, model viewer converts GLB to USDZ so we can host a singular model (praisebe), but I would also like to append parameters to the USDZ in order to insert custom buttons in quick look, see below:

https://developer.apple.com/documentation/arkit/adding_an_apple_pay_button_or_a_custom_action_in_ar_quick_look

Any advice?

**Comment by elalish** [comment]
Yes, I would like this too. Unfortunately it seems Safari has a bug blocking us: #2405.

**Reply by elalish** [reply]
@grorg Another Quick Look request.
